WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -90,10 +101,14 @@ - (instancetype)initWithPost:(AbstractPost *)aPost shouldHideStatusBar:(BOOL)sho
     return self;
 }
 
+#pragma mark - UIViewController
+
 - (void)viewDidLoad
 {
     [super viewDidLoad];
 
+    [self setupReachability];
```

Comment:
@diegoreymendez this is minor, but do you think the `setupReachability` method should be called at the end of this `viewDidLoad` implementation? If the `Reachbility` implementation were to ever call the `reachableBlock` synchronously after starting the notifier, it would call the `internetIsReachableAgain` method before everything has been initialized further down `viewDidLoad` implementation.